adoc/admin-updates.adoc

```diff
@@ -1,12 +1,19 @@
-== Control Plane
+= Handling updates
 
-In order to get an overview, which cluster-wide updates of the control plane are available, you can run:
+== Kubernetes components
```

Comment:
I would probably also mention the work `Platform` here, so we can easily distinguish between Platform and OS upgrades?

Reply:
SUSE CaaS Platform contains both the kubernetes components and the base OS. Here I'm referencing only the components because that's what the skuba commands are covering.